src/Microsoft.DotNet.Interactive/FrontendEnvironment.cs

```diff
@@ -1,11 +1,17 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Threading.Tasks;
 
 namespace Microsoft.DotNet.Interactive
 {
-    public abstract class FrontendEnvironment        
+    public abstract class FrontendEnvironment
     {
         public bool AllowStandardInput { get; set; }
+
+        public virtual Task ExecuteClientScript(string code, KernelInvocationContext context)
```

Comment:
This method might not be needed here; I think it's only ever consumed directly off of `HtmlNotebookFrontendEnvironment`.

Reply:
Just kidding, it's used in `JavascriptKernel`.  We can either leave the method here, or check for a strong type in the JS kernel.